g3data/g3data-window.c

```diff
@@ -270,10 +270,11 @@ static void g3data_window_file_save_as (GtkAction *action, G3dataWindow *window)
 
     dialog = gtk_file_chooser_dialog_new ("Save As...",
                                           GTK_WINDOW (window),
-                                          GTK_FILE_CHOOSER_ACTION_OPEN,
+                                          GTK_FILE_CHOOSER_ACTION_SAVE,
                                           GTK_STOCK_CANCEL, GTK_RESPONSE_CANCEL,
                                           GTK_STOCK_SAVE, GTK_RESPONSE_ACCEPT,
                                           NULL);
+    gtk_file_chooser_set_current_name(dialog, "untitled.txt");
```

Comment:
When compiling, I get the warning `warning: passing argument 1 of ‘gtk_file_chooser_set_current_name’ from incompatible pointer type`, because `gtk_file_chooser_set_current_name` expects a `GtkFileChooser *` as the first argument. Can you fix the warning (by casting via `GTK_FILE_CHOOSER (dialog)`)?

Reply:
Yes, GtkFileChooser is a base class for GtkFileChooserDialog and adding GTK_FILE_CHOOSER (dialog) fixes the warning.
```suggestion
    gtk_file_chooser_set_current_name(GTK_FILE_CHOOSER(dialog), "untitled.txt");
```